internal/sirius/add_contact.go

```diff
@@ -0,0 +1,66 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type ContactDetails struct {
```

Comment:
Can you rename this to `Contact`? `ContactDetails` suggests it is the contact details for a deputy, as opposed to a contact.